vaccine/vaccine_reg_whatsapp.py

```diff
@@ -101,6 +102,9 @@ async def state_timeout(self):
     async def state_age_gate(self):
         self.user.answers = {}
 
+        if random.random() <= config.THROTTLE_PERCENTAGE * 100:
```

Comment:
I don't think this is correct - `random.random()` always returns a value less than `1`

Reply:
Whoops, thanks